pkg/logentry/stages/util.go

```diff
@@ -64,6 +65,19 @@ func convertDateLayout(predef string, location *time.Location) parser {
 		}
 	case "Unix":
 		return func(t string) (time.Time, error) {
+			if strings.Count(t, ".") == 1 {
+				split := strings.Split(t, ".")
```

Comment:
should you return an error if `len(split) != 2` ?

Reply:
The split is only performed if there is exactly one dot in the string...
Not sure if we should return an error if there are multiple dots in the string?

Reply:
My point is with a single dot you could have one or two value in the split. One would crash here.

Reply:
right, good point

Reply:
added..